modules/nf-core/epang/split/main.nf

```diff
@@ -0,0 +1,37 @@
+process EPANG_SPLIT {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda (params.enable_conda ? "bioconda::epa-ng=0.3.8" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/epa-ng:0.3.8--h9a82719_1':
+        'quay.io/biocontainers/epa-ng:0.3.8--h9a82719_1' }"
+
+    input:
+    tuple val(meta), path(refaln)
+    path  fullaln
+
+    output:
+    tuple val(meta), path("query.fasta.gz"), emit: query
+    path  "reference.fasta.gz"             , emit: reference
+    path "versions.yml"                    , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    epa-ng \\
+        $args \\
+        --split $refaln $fullaln
+
+    gzip query.fasta reference.fasta
```

Comment:
Could you use the prefix in the naming of these files, so that it can be used by pipeline developers to determine output paths?

Reply:
As far as I can tell no, the output file names are hardcoded.

Reply:
I mean that you can rename the files to include the prefix

Reply:
Right. Done.